docs/input/docs/localization-howto.md

```diff
@@ -0,0 +1,35 @@
+Description: Localization In ChocolateyGUI
+---
+
+## Basics
+ChocolateyGUI uses `.resx` files for all its strings. 
+Basically it detects which [CultureInfo](https://msdn.microsoft.com/en-us/library/system.globalization.cultureinfo(v=vs.110).aspx) 
+best fits your current system and chooses the correct `.resx` file automagically.
+
+The most important thing to remember is: each language has it's own `Resources.lang.resx`, where `lang` is referring to the language culture name.
```

Comment:
Is there a way in these strings we can make it clearer that `lang` should be swapped out? Like `Resources.<lang>.resx` or some other not valid as a file name indicator?